themes/10up-theme/includes/blocks.php

```diff
@@ -93,12 +93,12 @@ function blocks_editor_styles() {
  * Filters the registered block categories.
  *
  * @param array  $categories Registered categories.
- * @param object $post       The post object.
+ * @param object $block_editor_context The current block editor context.
  *
  * @return array Filtered categories.
  */
-function blocks_categories( $categories, $post ) {
-	if ( ! in_array( $post->post_type, array( 'post', 'page' ), true ) ) {
+function blocks_categories( $categories, $block_editor_context ) {
+	if ( ! in_array( $block_editor_context->post->post_type, array( 'post', 'page' ), true ) ) {
```

Comment:
Does this account for custom post types?

Reply:
@ryanwelcher It doesn't look like it. Looking at the code on `trunk` and it has [this same guard clause](https://github.com/10up/wp-scaffold/blob/7f573eaaf2294b7c88a5034b7ba76766fcc57ca0/themes/10up-theme/includes/blocks.php#L101) for Posts and Pages. I'm guessing the original author assumed most blocks are made for built-in post types and doesn't want to clutter up custom post types. FWIW, I'm fine removing it.

Reply:
@ryanwelcher It doesn't look like it. Looking at the code on trunk and it has this same guard clause for Posts and Pages. I'm guessing the original author assumed most blocks are made for built-in post types and doesn't want to clutter up custom post types. FWIW, I'm fine removing it.

Reply:
@claytoncollie I agree that we should remove this check and have it apply to all editor contexts. Can you make that change?

Reply:
All done.

Reply:
All done @fabiankaegy